easybuild/easyconfigs/k/Kent_tools/Kent_tools-20200630-linux.x86_64.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'BinariesTarball'
+
+name = 'Kent_tools'
+version = '20200630'
+versionsuffix = '-linux.x86_64'
+
+homepage = 'https://genome.cse.ucsc.edu/'
+description = """Kent tools: collection of tools used by the UCSC genome browser."""
+
+toolchain = SYSTEM
+
+# Check the last modified date at http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64
+# Then and pack into tarball with:
+#   mkdir Kent_tools
+#   rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/ Kent_tools
+#   tar cfvz Kent_tools-20200630.tar.gz Kent_tools
+#
+sources = [SOURCE_TAR_GZ]
+checksums = ['1016adaadafd0d70421b9b99c0715d968f1dfabd6a3a20ca6dce7327d67f1b39']
```

Comment:
@verdurin Since tarball is self-made, including a checksum doesn't make sense here (other that create the tarball manually will get a different checksum)

Reply:
@boegel you're suggesting this should be added to the ignore list instead?

Reply:
@verdurin It's already there, see https://github.com/easybuilders/easybuild-easyconfigs/blob/c99b207cb3691506868827d5b27181d6df252dec/test/easyconfigs/easyconfigs.py#L526-L528

But perhaps #11121 is a better way forward?
I only recently discovered that building those tools from source is an option...